tests/roots/test-ext-javadoctest-maven/index.rst

```diff
@@ -0,0 +1,27 @@
+Test Java With Third Dependencies Handled by Maven
+==================================================
+
+In case we need to test documentation for projects that consume Java native libraries and
+Third party Java dependencies, then, is needed to configure/add these dependencies thru a
+dependency management suchs as Maven/Gradle, in this case initially we are offering support
+for Maven.
+
+In this case is needed to define by `conf.py` a flavor with `java_with_maven`, and also define
```

Comment:
This paragraph needs to be more precise: You should explain the new options using the `.. confval::` directive (e.g., what is `javadoc_options`, etc). Again this paragraph should be moved to the documentation and not in the testroot (I know that this is the last milestone of your tasks but keep in mind this comment).